auth/api/iam/api.go

```diff
@@ -71,9 +71,10 @@ type Wrapper struct {
 	policyBackend policy.PDPBackend
 	templates     *template.Template
 	storageEngine storage.Engine
+	keyStore      crypto.KeyStore
 }
 
-func New(authInstance auth.AuthenticationServices, vcrInstance vcr.VCR, vdrInstance vdr.VDR, storageEngine storage.Engine, policyBackend policy.PDPBackend) *Wrapper {
+func New(authInstance auth.AuthenticationServices, vcrInstance vcr.VCR, vdrInstance vdr.VDR, storageEngine storage.Engine, policyBackend policy.PDPBackend, keyStore crypto.KeyStore) *Wrapper {
```

Comment:
changes not needed when handled by wallet

Reply:
Gone